### ISSUE TITLE
linkerd install ignores --identity-external-issuer when --ha is used

### ISSUE THREAD

**Issue by erwbgy** [linkerd/linkerd2#5456]
### What is the issue?

The `linkerd install` command ignores `--identity-external-issuer` and generates Secret values when `--ha` is passed as well.

### How can it be reproduced?

```
$ linkerd install --identity-external-issuer --disable-heartbeat --linkerd-cni-enabled --restrict-dashboard-privileges | \
grep -B3 linkerd-identity-issuer
      volumes:
      - name: identity-issuer
        secret:
          secretName: linkerd-identity-issuer

$ linkerd install --ha --identity-external-issuer --disable-heartbeat --linkerd-cni-enabled --restrict-dashboard-privileges | \
grep -B3 linkerd-identity-issuer
kind: Secret
apiVersion: v1
metadata:
  name: linkerd-identity-issuer
--
      volumes:
      - name: identity-issuer
        secret:
          secretName: linkerd-identity-issuer
```

### Environment

- Kubernetes Version: 1.18.11
- Cluster Environment: kubeadm
- Host OS: RHEL 8.2
- Linkerd version: stable-2.9.1

**Comment by cpretzer** [comment]
Thanks for this report @erwbgy There were a few `ha` issues fixed in the latest edge release:

https://github.com/linkerd/linkerd2/pull/5399
https://github.com/linkerd/linkerd2/pull/5358

I'll see if I can repro this with the latest edge release

**Comment by erwbgy** [comment]
Thanks @cpretzer, it is indeed fixed in edge-20.12.4 - sorry for the noise.

**Comment by cpretzer** [comment]
Thanks for confirming @erwbgy !
